app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterialForm.java

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.powerpoles_material;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.ImageListQuestAnswerFragment;
+
+public class AddPowerPolesMaterialForm extends ImageListQuestAnswerFragment
+{
+	private final OsmItem[] TYPES = new OsmItem[] {
+			new OsmItem("wood", R.drawable.power_pole_wood, R.string.quest_powerPolesMaterial_wood),
+			new OsmItem("metal", R.drawable.power_pole_metal, R.string.quest_powerPolesMaterial_metal),
```

Comment:
It's more commonly tagged as `steel`, not `metal`. Which is more correct, IMO.

Reply:
And as https://poeditor.com/projects/po_edit?id=97843&term=67125651&id_language=127#showComments67125651 points out

> Probably misleading that it says metal there at all but I've checked the code and this option does correspond to the tag 'steel', not 'metal', being added.